apps/fabric-website/src/pages/Styles/LayoutPage/LayoutPage.tsx

```diff
@@ -146,7 +146,7 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
                 add up to 12 for each device size.
               </p>
               <p>
-                Newer versions of Fabric require the <code>dir</code> attribute to be set to specify how the content should be rendered
+                Newer versions of Fluent UI require the <code>dir</code> attribute to be set to specify how the content should be rendered
```

Comment:
Everything on this page needs to be Fabric Core. Probably also good to add a work item for scrubbing other pages under Styles to ensure we're clear about what's Fabric Core and what's Fluent UI.